include/configs/pistachio_bub.h

```diff
@@ -307,18 +304,25 @@
 
 #endif
 
+/*
+ * Openwrt have squashfs rootfs but legacy and other variant uses ubifs
+ * rootfs. To make all compatible dont pass root= for kernel booting
+ * from kernel ubi volume. For others, pass root= to correctly identify
+ * rootfs filesystem.
```

Comment:
This doesn't explain why this works, nor does the commit message. I would expect the commit message to go into detail of the different boot scenarios this supports.